packages/opentelemetry-node/lib/instrumentations.js

```diff
@@ -59,6 +60,7 @@
  */
 
 /* eslint-disable prettier/prettier */
+const {OpenAIInstrumentation} = require('@elastic/opentelemetry-instrumentation-openai');
```

Comment:
I guess the alphabetical order is per package scope & name? if this goes upstream we should move it down then?

Reply:
I guess. That's why I put it at the top. I don't have a preference though.